File: 1720-crawler-log-folder/1720-crawler-log-folder.java
class Solution {
    private int pos(String log, int ops) {
        return log.charAt(1) == '.' ? ops > 0 ? -1 : 0 : log.charAt(0) == '.' ? 0 : 1;
    }
    public int minOperations(String[] logs) {
        int ops = 0;
        for (String log: logs) ops += pos(log, ops);
        return ops;
    }
}

File: 1720-crawler-log-folder/1720-crawler-log-folder.cpp
class Solution {
private:
    int ops;
    int pos(string log) {
        return log[1] == '.' ? ops > 0 ? -1 : 0 : log[0] == '.' ? 0 : 1;
    }
public:
    int minOperations(vector<string>& logs) {
        ops = 0;
        for(string log: logs) ops += pos(log);
        return ops;
    }
};